### ISSUE TITLE
NormalMap textures working incorrectly

### ISSUE THREAD

**Issue by andeliseev** [pschraut/UnityTexture2DArrayImportPipeline#8]
If add texture with texture type Normal Map, texture2darray format would be wrong

**Comment by pschraut** [comment]
Can you please add what Unity version you use and a step-by-step list how I can reproduce the problem?

**Comment by andeliseev** [comment]
It’s reproduce in all unity version. I think it is not you package problem, but internal unity bug. It swizzle or doing something with normals map in editor. To reproduce you can switch target platform to android and make texture format ASTC. Normal map in array will be looking washed out like it double linear. And   If you test it quickly in shader graph result will be wrong

**Comment by Jimbologo** [comment]
I believe its due to the textures not being set to non-SRGB texture. We need an option in the inspector to be able to set the textures to SRGB or non-SRGB (Linear Space).

**Comment by pschraut** [comment]
I've looked into the issue, but I'm unable to reproduce it.

The importer already handles sRGB/Linear formats, I've double-checked that.

A problem seems to be with the normal-map preview in the Inspector window. It appears reddish instead of the usual bluish colors associated with normal-maps.

However, when I use the normal-map TextureArray to shade eg a Cube, it looks correct to me.

**Comment by Jimbologo** [comment]
> I've looked into the issue, but I'm unable to reproduce it.
> 
> The importer already handles sRGB/Linear formats, I've double-checked that.
> 
> A problem seems to be with the normal-map preview in the Inspector window. It appears reddish instead of the usual bluish colors associated with normal-maps.
> 
> However, when I use the normal-map TextureArray to shade eg a Cube, it looks correct to me.

Yeah i've had a double look into this. i was wrong and you are correct it is just the preview showing up more reddish/orange than the actual texture which is pretty strange and yeah what it outputs is correct also. Its not really an issue
